src/telegram-bot-api-client.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import {logger} from './logger.js';
+
+class TelegramBotApiClient {
+    constructor(botAPI, botToken) {
+        this.botAPI = botAPI;
+        this.botToken = botToken;
+        this.http = axios.create({
+            baseURL: this.botAPI + this.botToken
+        })
+    }
+
+    async sendMessage(chatID, message) {
+        const body = {
+            chat_id: chatID,
+            text: message
+        }
+
+        try {
+            await this.http.post('/sendMessage', body);
+        } catch (err) {
+            logger.error(`Request to sendMessage failed with error`, err);
+            throw err
```

Comment:
I don't think we want to throw the error here, otherwise we might get an uncaught rejected promise

Reply:
actually, maybe better to keep this and handle / catch it at the caller (`TelegramNotifier`) level?

Reply:
yes, it is better to handle it in TelegramNotifier. Also, we have a beacon API client implemented the same way, so let's keep it aligned.